docs/source/index.rst

```diff
@@ -1,19 +1,59 @@
 =========
-Estimagic
+estimagic
 =========
 
-Estimagic is a Python package that helps to build high-quality and user friendly
-implementations of (structural) econometric models. It is designed with large structural
-models in mind, but also "scales down" to simple cases.
+estimagic is a Python package that provides high-quality and user-friendly tools
```

Comment:
The readme and the introductory text (including the dashboard gif) should be aligned. Can you also add the changes to the readme?